site/app/views/submission/HomeworkView.php

```diff
@@ -97,6 +119,8 @@ public function showGradeable($gradeable, $days_late) {
     <script type="text/javascript">
         $(document).ready(function() {
             var cookie = document.cookie;
+            student_ids = $student_ids;
```

Comment:
If you're using PHP functions, please use the `{$student_ids}` just to keep it consistent through the code base.